docs/configuring-playbook-bot-buscarron.md

```diff
@@ -0,0 +1,75 @@
+# Setting up Buscarron (optional)
+
+The playbook can install and configure [buscarron](https://gitlab.com/etke.cc/buscarron) for you.
+
+It's a bot you can use to setup **your own helpdesk on matrix**
+It's a bot you can use to send any form (HTTP POST, HTML) to a (encrypted) matrix room
+
+## Registering the bot user
+
+By default, the playbook will set up the bot with a username like this: `@buscarron:DOMAIN`.
```

Comment:
Shouldn't we use `@bot.buscarron:DOMAIN` to be consistent with the rest of the playbook' bots?